### ISSUE TITLE
Create an event in "Gummibären" calendar if anyone attends

### ISSUE THREAD

**Issue by BjoernPetersen** [OpenAlcoholics/regular_dicers_bot#4]
Depends on #3

**Comment by Chabare** [comment]
We have a recurring event anyway. I don't really feel that this is necessary.

**Comment by BjoernPetersen** [comment]
This would only create the event if it actually takes place. I think it would actually be an enhancement, but obviously not a priority at all.

**Comment by Chabare** [comment]
This 506eeb000faca2e9b2fc8fc6c6d329a1402892a9 should do it (not updated yet). Will update before 16:00 today.

**Comment by BjoernPetersen** [comment]
I did not expect that.

**Comment by Chabare** [comment]
Me neither
